core/server/worker/src/test/java/alluxio/worker/block/PinListSyncTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.ClientContext;
+import alluxio.conf.Configuration;
+import alluxio.master.MasterClientContext;
+import alluxio.worker.file.FileSystemMasterClient;
+
+import com.google.common.collect.ImmutableSet;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Set;
+
+public class PinListSyncTest {
+
+  // context for creating FileSystemMasterClient
+  private final MasterClientContext mContext = MasterClientContext.newBuilder(ClientContext.create(
+      Configuration.global())).build();
+
+  private TestBlockWorker mBlockWorker;
+
+  @Before
+  public void before() {
+    mBlockWorker = new TestBlockWorker();
```

Comment:
This is redundant. Initialize where it is declared.

Reply:
I believe JUnit creates a new instance of the test class for every `@Test`. So initializing things in constructor is the same as in a `@Before` method. [reference](https://stackoverflow.com/q/6094081)

Reply:
Yeah that is actually the case. I wonder why they would like doing this though...